addons/sale_timesheet/migrations/11.0.1.0/post-migration.py

```diff
@@ -47,12 +48,72 @@ def migrate_project_sale_line(env):
         """, (AsIs(service_column),))
 
 
+def map_track_service(env):
+    """"Map values for old `track_service` field (copied in sale pre-migration)
+    according this mapping:
+
+    track_service           service_tracking
+    -------------           ----------------
+    'manual'		         'no'
+    'timesheet'		        if not project_id: 'task_new_project'
+                            if project_id: 'task_global_project'
+    'task'			        'project_only'
+
+    Project field depends on company, so this is applicable as soon as one
+    company has any project set.
+    """
+    openupgrade.map_values(
+        env.cr,
+        openupgrade.get_legacy_name('track_service'),
+        'service_tracking', [
+            ('manual', 'no'),
+            ('timesheet', 'project_only'),
```

Comment:
you put above that `timesheet` is mapped to `task_new_project` or `task_global_project` but not `project_only`

Reply:
sorry, incorrect docstring, but proper scripts. Fixing docs.

Reply:
Done